Makefile

```diff
@@ -92,6 +92,9 @@ PAUSEBINRELPATH := bin/pause
 
 GLOBALLOGPATH := $(PKGLIBDIR)/runtime/runtime.log
 
+DEFDEFVCPUS := 1
```

Comment:
I'm not clear on the "double DEF" here. Couldn't this just be `DEFAULTVCPUS` and `DEFAULTMEMSZ`?